src/components/OverflowMenuItem/OverflowMenuItem.js

```diff
@@ -7,6 +7,7 @@ const OverflowMenuItem = ({
   itemText,
   hasDivider,
   isDelete,
+  isDisabled,
```

Comment:
Can we change this to `disabled` so it follows the pattern that other components are using.